.github/pull_request_template.md

```diff
@@ -6,13 +6,15 @@
 
      You can use the CI build or a local environment to check your changes are working as expected.
      
-     If you have questions at any time, please contact the Website Working Group at
+     If you have questions at any time, please contact the Website & Content Working Group at
      https://matrix.to/#/#matrix.org-website:matrix.org -->
 
-<!-- Please describe what you added, and add a screenshot or video if possible.
+<!-- ------------------------------------------------------------------------------------------
+     Please describe what you added here, and add a screenshot or video if possible.
      That makes it easier to understand the change. -->
 
-<!-- Please don't remove this checklist from your PR, as it is useful for reviewers, too. -->
+<!-- ------------------------------ DO NOT WRITE BELOW THIS LINE ------------------------------
```

Comment:
I would add this after the checklist, not before. I know my two brain cells could take time to fire at each other and wonder if this is a checklist for the reviewer